examples/README.md

```diff
@@ -10,17 +10,10 @@ The following examples are included:
 * `KeyValueViewExample` - demonstrates the usage of the `org.apache.ignite.table.KeyValueView` API
 * `SqlJdbcExample` - demonstrates the usage of the Apache Ignite JDBC driver.
 * `RebalanceExample` - demonstrates the data rebalancing process.
+* `VolatilePageMemoryStorageExample` - demonstrates the usage of the PageMemory storage engine configured with an in-memory data region.
 
-To run the `RebalanceExample`, refer to its JavaDoc for instructions.
+Before running the examples, read abaut [cli](https://ignite.apache.org/docs/3.0.0-alpha/ignite-cli-tool).
 
-To run any other example, do the following:
-1. Import the examples project into you IDE.
-2. Start a server node using the CLI tool:
-   ```
-   ignite node start --config=$IGNITE_HOME/examples/config/ignite-config.json my-first-node
-   ```
-3. Initialize the node using the following command:
-   ```
-   ignite cluster init --node-endpoint 'localhost:10300' --meta-storage-node 'my-first-node' --cluster-name 'ignite-cluster'
-   ```
-4. Run the preferred example in the IDE.
+To run the examples, refer to their JavaDoc for instructions.
+
+If errors occur at the cluster initialization step, it is recommended to delete the `ignite-work` directory and repeat the instruction to run the example.
```

Comment:
I think this is quite a misleading sentence - it makes an impression that we may fail for no reason and you need to remove everything. I think it would be better to specify what error are you talking about and why it happens

Reply:
Remove it.